docs/get-started/deployment-and-usage.md

```diff
@@ -94,22 +96,23 @@ to the metadata of NodeFeatureDiscovery object above.
 The template specs provided in the repo can be used directly:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/node-feature-discovery/{{ site.release }}/nfd-master.yaml.template
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/node-feature-discovery/{{ site.release }}/nfd-worker-daemonset.yaml.template
+make deploy
```

Comment:
IMO, with kustomize it must be possible to `kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deploy?ref=<tag|branch>`. `make deploy` is not nice because the repo must be cloned first.

Reply:
we should be able for sure, but from their new location `templates/nfd-master` && `templates/nfd-worker` I can do an extra thing for the `scripts/prepare-release` that will leave the yamls ready to be used with the <tag|branch> version

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery/pull/558/commits/fc651fc9b4a8e8d949efdeb8ccc0d6f2b02510d3 aim to fix this

Reply:
Does https://github.com/kubernetes-sigs/node-feature-discovery/commit/fc651fc9b4a8e8d949efdeb8ccc0d6f2b02510d3 address your comment @mythi ?

Reply:
@ArangoGutierrez what is the reason to generate `deploy.yaml`. With `-k`, kubectl runs kustomize so generating static `deploy.yaml` seems unnecessary to me.

Reply:
I see what you mean, will fix